service/frontend/adminHandler.go

```diff
@@ -445,10 +445,15 @@ func (adh *adminHandlerImpl) deleteWorkflowFromHistory(
 			logger.Error("Cannot decode thrift object", tag.Error(err))
 			continue
 		}
+		domainName, err := adh.Resource.GetDomainCache().GetDomainName(mutableState.ExecutionInfo.DomainID)
```

Comment:
this could be `adh.GetDomainCache().GetDomainName (...)`
to make it consistent with other references